From cc5def8300fa79ff6d14ac72bc38d0c71e6817c1 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 10 May 2023 13:39:25 +0200
Subject: [PATCH] arm64: defconfig: enable STM32 DMA3 in addons fragment

Enable STM32 DMA3 support on STM32 arm64 platforms in addons fragment.

Change-Id: Iae2fb7787220d162cab2a4c8de5055aa784300e8
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -13,6 +13,7 @@ CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REMOTEPROC=y
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
+CONFIG_STM32_DMA3=y
 CONFIG_STM32_RPROC=y
 CONFIG_STM32_LPTIMER_CNT=m
 CONFIG_STM32_TIMER_CNT=m
